policyengine_us/variables/gov/states/pa/tax/income/forgiveness/pa_tax_forgiveness_rate.py

```diff
@@ -33,9 +33,6 @@ def formula(tax_unit, period, parameters):
         )
         excess = eligibility_income - eligibility_income_increment
         forgiveness_increment = p.rate_increment
-        if parameters(period).gov.contrib.nber.taxsim35_emulation:
-            increments = excess / forgiveness_increment
-        else:
-            increments = np.ceil(excess / forgiveness_increment)
+        increments = np.ceil(excess / forgiveness_increment)
```

Comment:
@martinholmer can I just check this is correct: if this branch was emulating taxsim, why did it change the code to act as in the non-emulation parameter scenario?